src/main/kotlin/no/nav/familie/klage/kabal/domain/OversendtKlage.kt

```diff
@@ -114,7 +113,6 @@ enum class LovKilde(override val id: String, override val navn: String, override
         "Forskrift om beregning av uføretrygd etter EØS- avtalen 883/2004",
     ),
     GJENNOMFØRINGSFORORDNING_987_2007("19", "Gjennomføringsforordning 987/2007", "Gjennomføringsforordning 987/2007"),
-    EØS_FORORDNING_1408_71("20", "Forordning 1408/71", "Forordning 1408/71"),
```

Comment:
Det er nesten ingen av enum-verdiene i denne enumen som faktisk brukes. Kan vi rydde opp eller er det en grunn til at de ligger der?

Reply:
Alle enumene som ligger i den filen er kopiert inn direkte fra Kabal sitt kodeverk. Vi visste ikke hvilke vi ville få bruk for på den tiden de ble kopiert inn